test/regression.js

```diff
@@ -124,57 +115,48 @@ const readdirRecursive = async (absolute, relative = '') => {
   return result;
 };
 
+const width = 960;
+const height = 720;
 (async () => {
   try {
     const start = process.hrtime.bigint();
     const fixturesDir = path.join(__dirname, 'regression-fixtures');
     const list = await readdirRecursive(fixturesDir);
-    const originalFiles = new Map();
-    const optimizedFiles = new Map();
-    // read original and optimize
-    let failed = 0;
-    for (const name of list) {
+    // setup server
+    const server = http.createServer(async (req, res) => {
+      const name = req.url.slice(req.url.indexOf("/", 1));
+      let file;
       try {
-        const file = path.join(fixturesDir, name);
-        const original = await fs.promises.readFile(file, 'utf-8');
-        const result = optimize(original, { path: name, floatPrecision: 4 });
-        if (result.error) {
-          console.error(result.error);
-          console.error(`File: ${name}`);
-          failed += 1;
-        } else {
-          originalFiles.set(name, original);
-          optimizedFiles.set(name, result.data);
-        }
+        file = await fs.promises.readFile(
+          path.join(fixturesDir, name),
+          'utf-8',
+        );
       } catch (error) {
-        console.error(error);
-        console.error(`File: ${name}`);
-        failed += 1;
+        res.statusCode = 404;
+        res.end();
+        return;
       }
```

Comment:
Could you replace all of this with just:

```js
const file = await fs.promises.readFile(
  path.join(fixturesDir, name),
  'utf-8',
);
```

If it fails to read the file, we should know about it. Not silently respond with a blank page. I'd expect this to crash the tests.